holidays/countries/algeria.py

```diff
@@ -32,6 +32,7 @@ class Algeria(HolidayBase, InternationalHolidays, IslamicHolidays):
     # %s (estimated).
     estimated_label = tr("(تقدير) %s")
     supported_languages = ("ar", "en_US", "fr")
+    weekend = {FRI, SAT}
```

Comment:
But value is set in `_populate_public_holidays()`.

Reply:
I must've accidentally put it there while reviewing other entries, fixed.